### ISSUE TITLE
Cookie consent i18n and RTL

### ISSUE THREAD

**Issue by renevatium** [proversity-org/edx-platform#260]

**Comment by lidijarad** [comment]
Need to do:

- [x] Checkout en/django.po file from upstream/master (we never need to make changes to these files.)
- [x] Address colour contrast for background/text colour. So far, #007db8 fails WCAG AAA contrast test. (Need to make it darker to pass the test)
- [x] Make banner tabbable
- [x] Make button respond to onkeypress event.
- [x] What can we do about the default window div that uses aria-live?

**Comment by lidijarad** [comment]
Merged. PR accepted. Closing.
